src/Uno.UI/UI/Xaml/GridLength.cs

```diff
@@ -19,7 +19,7 @@ public partial struct GridLength : IEquatable<GridLength>
 
 		public bool IsStar { get { return GridUnitType == Xaml.GridUnitType.Star; } }
 
-		public double Value { get; private set; }
+		public double Value { get; internal set; }
```

Comment:
?

Reply:
Based on this:

https://github.com/microsoft/microsoft-ui-xaml/blob/bd98a75dabfe0ad0a8956834ea9da735781506e6/dev/NavigationView/NavigationView.cpp#L1756-L1758

But maybe can just create a new `GridLength`.

Reply:
Oh! Fair enough.

Reply:
Changed it anyway, seems not worth introducing mutability :-D